domains/gorciu.json

```
@@ -0,0 +1,10 @@
+{
+    "description": "My portfolio website",
+    "owner": {
+        "username": "gorciu-official",
+        "email": "gorciuyt@gmail.com",
```

```suggestion
        "email": "gorciuyt@gmail.com"
```
Remove the comma here, 'tis invalid formatting.